app/assets/stylesheets/welcome.scss

```diff
@@ -4,6 +4,15 @@
 
 body { padding-top: 50px; }
 
-h1, h2, p {
+h1, h2, {
+  text-align: center;
+}
+
+.col-md-8 {
+  float: none;
+  margin: auto;
+}
```

Comment:
Usually, you don't want to directly edit Bootstrap's classes, because it will make the edit global while you're trying to fix a special case. You could simply add another "semantic" class (ie. a meaningful, role-focused class) on the element, alongside `.col-md-8`, and style it instead: `.main-content`, etc.